packages/browser-destinations/src/destinations/upollo/identifyUser/index.ts

```diff
@@ -70,7 +70,21 @@ const identifyUser: BrowserActionDefinition<Settings, UpolloClient, Payload> = {
       customerSuppliedValues: payload.custom_traits ? toCustomValues(payload.custom_traits) : undefined
     }
 
-    void UpClient.track(userInfo)
+    try {
+      const result = await UpClient.assess(userInfo)
+      if (result.emailAnalysis && result?.emailAnalysis?.company?.name != '' && settings?.companyEnrichment) {
+        context.updateEvent('traits.company', {
+          name: result?.emailAnalysis?.company?.name,
+          industry: result?.emailAnalysis?.company?.industry,
+          employee_count: Math.max(
+            result.emailAnalysis?.company?.companySize?.employeesMax,
+            result.emailAnalysis?.company?.companySize?.employeesMin
+          )
+        })
+      }
+    } catch (e) {
+      console.log(e)
+    }
```

Comment:
I dont think we should be catching and logging errors to the console in production. I understand this is great during development, but maybe we can take it out now.

```suggestion
    const result = await UpClient.assess(userInfo)
    if (result.emailAnalysis && result?.emailAnalysis?.company?.name != '' && settings?.companyEnrichment) {
      context.updateEvent('traits.company', {
        name: result?.emailAnalysis?.company?.name,
        industry: result?.emailAnalysis?.company?.industry,
        employee_count: Math.max(
          result.emailAnalysis?.company?.companySize?.employeesMax,
          result.emailAnalysis?.company?.companySize?.employeesMin
        )
      })
    }
```